src/core/lib/iomgr/tcp_server_posix.c

```diff
@@ -70,6 +71,8 @@
 
 #define MIN_SAFE_ACCEPT_QUEUE_SIZE 100
 
+int grpc_tcp_server_posix_expand_wildcard_addr = 0;
```

Comment:
Craig, sounds good.

There are also some bug fixes in this PR (tcp_server_get_port_fd()) that I will split into a separate PR.